.gitignore

```diff
@@ -10,7 +10,7 @@ coverage
 
 # Trailing dotted files
 !.all-contributorsrc
-!.babelrc
+!.babelrc.js
```

Comment:
I probably have not committed this file before and didnt notice that it was ignored 🙈